content/en/docs/setup/getting-started/index.md

```diff
@@ -8,11 +8,13 @@ aliases:
 keywords: [getting-started, install, bookinfo, quick-start, kubernetes]
 ---
 
-To get started with Istio, just follow these three steps:
+To get started with Istio, just follow these steps:
```

Comment:
Please put this back to 3 steps. What's next and uninstall are not steps to get started.